client/src/pages/Review/CoursePanel.tsx

```diff
@@ -121,12 +121,11 @@ const CoursePanel = ({
 
         <section
           className={classnames(
-            "p-4 mr-2",
+            "p-4 mt-2 mr-2",
             "w-full md:w-1/2",
-            "h-screen",
+            "md:h-screen",
```

Comment:
On small screens, nothing it restricting the content height so it remains in flow and scrolls with the rest of the page